### PR TITLE
fwb: improvments

### PR DESCRIPTION
**Pull request by gotenksIN**
* Allow us to change max notif icons
* Let legacy apps scale fully to the display's aspect ratio

### CODE REVIEW

File: core/res/res/values/yaap_symbols.xml [hunk 1]
```diff
@@ -29,4 +29,7 @@
     <!-- Device has OnePlus haptic motor -->
     <java-symbol type="bool" name="config_hasOnePlusHapticMotor" />
 
+  <!-- Max visible notification icons -->
```
Comment (poad42) [inline core/res/res/values/yaap_symbols.xml]: phix tabbing before @idoybh goes crazy.

File: packages/SystemUI/src/com/android/systemui/statusbar/phone/NotificationIconContainer.java [hunk 2]
```diff
@@ -135,8 +135,10 @@ public AnimationFilter getAnimationFilter() {
         }
     }.setDuration(CONTENT_FADE_DURATION);
 
-    private static final int MAX_VISIBLE_ICONS_ON_LOCK = 5;
-    public static final int MAX_STATIC_ICONS = 4;
+    private final int MAX_VISIBLE_ICONS_ON_LOCK =
+            getResources().getInteger(com.android.internal.R.integer.config_maxVisibleNotificationIconsWhenDark);
```
Comment (poad42) [inline packages/SystemUI/src/com/android/systemui/statusbar/phone/NotificationIconContainer.java]: this can be one lined

**Review by poad42 — CHANGES_REQUESTED** [review]

**Comment by idoybh** [comment]
About the legacy apps commit. It should be per app based. Some apps may not work well if you force em to change ratio

File: packages/SystemUI/src/com/android/systemui/statusbar/phone/NotificationIconContainer.java (hunk 2 above)
Comment (idoybh) [reply packages/SystemUI/src/com/android/systemui/statusbar/phone/NotificationIconContainer.java]: this is already one lined

**Comment by gotenksIN** [comment]
so no-go on the legacy app one?

File: core/res/res/values/yaap_symbols.xml (hunk 1 above)
Comment (gotenksIN) [reply core/res/res/values/yaap_symbols.xml]: done

File: packages/SystemUI/src/com/android/systemui/statusbar/phone/NotificationIconContainer.java (hunk 2 above)
Comment (gotenksIN) [reply packages/SystemUI/src/com/android/systemui/statusbar/phone/NotificationIconContainer.java]: and done

**Comment by poad42** [comment]
the legacy app looked fine

**Comment by idoybh** [comment]
> the legacy app looked fine

It's not fine until we have an option to only enable for apps that work ok with that 
Omni had a commit for that back in 10. We can look into forward porting it

**Comment by poad42** [comment]
ah

**Comment by poad42** [comment]
the other one is fine so merging and closing pr

File: core/res/res/values/yaap_symbols.xml [hunk 3]
```diff
@@ -29,4 +29,7 @@
     <!-- Device has OnePlus haptic motor -->
     <java-symbol type="bool" name="config_hasOnePlusHapticMotor" />
 
+   <!-- Max visible notification icons -->
+   <java-symbol type="integer" name="config_maxVisibleNotificationIcons" />
+   <java-symbol type="integer" name="config_maxVisibleNotificationIconsWhenDark" />
```
Comment (idoybh) [inline core/res/res/values/yaap_symbols.xml:34]:
```suggestion
    <!-- Max visible notification icons -->
    <java-symbol type="integer" name="config_maxVisibleNotificationIcons" />
    <java-symbol type="integer" name="config_maxVisibleNotificationIconsWhenDark" />
```

**Review by idoybh — CHANGES_REQUESTED** [review]

File: core/res/res/values/yaap_symbols.xml (hunk 3 above)
Comment (idoybh) [reply core/res/res/values/yaap_symbols.xml:34]: grrrrrr @poad42 I will f-push to fwb

**Pull request merged**